src/models/Shape.js

```diff
@@ -1,11 +1,16 @@
-import Layer, { layerOptions } from './Layer'
+import Model, { modelOptions } from './Model'
 
 export const shapeOptions = {
-  ...layerOptions
 }
 
-export default class Shape extends Layer {
+export const shapeAttrs = {
+}
+
+export default class Shape extends Model {
   getOptions() {
     return shapeOptions
   }
+  getAttrs() {
```

Comment:
options are things the user changes. Attrs are things that don't change (for a given model/shape). The words Option and Attr comes up a lot, so I would prefer something shorter than frozenAttrs and protectedAttrs. Is there another word for Attrs that makes it sound like they are constant for a specific shape? Attributes sounded permanent to me. Especially because they come from the models. Maybe that is due to my particular coding experience.

As an example, FancyText has `usesFonts: true`. That will never change. The other shapes have `usesFonts: false`. Any layers that use the FancyText Model will use fonts, and that is attributed to the model, and permanent (user can't change that).

I also changed it so the Attrs don't end up in the object description. They are only accessed by finding the current model, and then querying that class for the Attrs. If you inspect the redux state, those attrs aren't in there.

Reply:
ok, I'm fine with this as-is.